src/TLuaInterpreter.cpp

```diff
@@ -7690,9 +7691,12 @@ int TLuaInterpreter::createMapLabel(lua_State* L)
             }
         }
     }
+    if (args > 15) {
```

Comment:
***15*** arguments - ouch! If we ever redo the API we are going to have to think about this! :astonished:

Reply:
Agreed